### ISSUE TITLE
Kitty

### ISSUE THREAD

**Issue by aleksandersumowski** [numToStr/Navigator.nvim#21]
Would it be possible to support the inbuilt Kitty terminal multiplexer?

**Comment by numToStr** [comment]
Yes, definitely. You can also experiment with it in the userland https://github.com/numToStr/Navigator.nvim/wiki/Custom-Multiplexer once it is working then feel free to raise a PR.

Although I am not sure whether or not kitty support navigation from cli(?)
